src/Core/Grid/Definition/Factory/CartGridDefinitionFactory.php

```diff
@@ -0,0 +1,320 @@
+                            ->setIcon('zoom_in')
+                            ->setOptions([
+                                'route' => 'admin_carts_view',
+                                'route_param_name' => 'cartId',
+                                'route_param_field' => 'id_cart',
+                            ])
+                        )
+                        ->add((new SubmitRowAction('delete'))
+                            ->setName($this->trans('Delete', [], 'Admin.Actions'))
+                            ->setIcon('delete')
+                            ->setOptions([
+                                'route' => 'admin_carts_delete',
+                                'route_param_name' => 'cartId',
+                                'route_param_field' => 'id_cart',
+                                'accessibility_checker' => $this->deleteCartAccessibilityChecker,
+                                'confirm_message' => $this->trans(
+                                    'Delete selected item?',
+                                    [],
+                                    'Admin.Notifications.Warning'
+                                ),
+                            ])
+                        ),
+                ])
+            )
+        ;
+
+        if ($this->configuration->get('PS_GUEST_CHECKOUT_ENABLED')) {
+            $columns->addAfter('date_add', (new DataColumn('online'))
+                ->setName($this->trans('Online', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'online',
+                    'text_align' => 'center',
+                ])
+            );
+        }
+
+        $columns->addBefore('actions', (new DataColumn('shop_name'))
+            ->setName($this->trans('Shop', [], 'Admin.Global'))
+            ->setOptions([
+                'field' => 'shop_name',
+            ])
+        );
+
+        return $columns;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getFilters()
+    {
+        $filters = (new FilterCollection())
+            ->add((new Filter('id_cart', NumberType::class))
+                ->setTypeOptions([
+                    'attr' => [
+                        'placeholder' => $this->trans('Search ID', [], 'Admin.Actions'),
+                    ],
+                    'required' => false,
+                ])
+                ->setAssociatedColumn('id_cart')
+            )
+            ->add((new Filter('status', DatalistType::class))
+                ->setTypeOptions([
+                    'choices' => [
+                        $this->trans('Not placed', [], 'Admin.Orderscustomers.Feature') => CartStatusType::NOT_ORDERED,
+                        $this->trans('Abandoned cart', [], 'Admin.Orderscustomers.Feature') => CartStatusType::ABANDONED_CART,
+                    ],
+                    'attr' => [
+                        'placeholder' => $this->trans('Search Order ID / Status', [], 'Admin.Actions'),
```

Comment:
```suggestion
                        'placeholder' => $this->trans('Search order ID / status', [], 'Admin.Actions'),
```

Reply:
Please modify with _Search order ID_ then, following the recent comments, thanks!